src/test/crimson/seastore/transaction_manager_test_state.h

```diff
@@ -116,15 +117,18 @@ class EphemeralTestState {
 
   seastar::future<> randomblock_setup()
   {
-    auto config =
-      journal::CircularBoundedJournal::mkfs_config_t::get_default();
     rb_device.reset(new random_block_device::TestMemory(
```

Comment:
Create a factory method for `TestMemory` (probably better rename to `EphemeralRBMDevice`) that creates the test device by the cbjournal and data area sizes. Probably cleaner to define and use the `TEST_BLOCK_SIZE` in `TestMemory` for all the tests.

Reply:
Done.